app/about/faq/text.tsx

```diff
@@ -0,0 +1,37 @@
+import Link from "next/link";
```

Comment:
I don't know what about this file makes the Github coloring lose it's mind but sorry about that. All of this copy is pulled from the serotracker.com FAQ section but I removed anything that was super CoV2 related or references documents which made sense for CoV2 and didn't have a arbovirus equivalent and wouldn't make sense since we don't have CoV2 yet and added some arbovirus related language.

I'm not super concerned about the copy here, I think it's reasonable enough for now. This is behind a feature flag, we can massage the copy a bit later.

Reply:
Yeah this is a good idea. I went back to my first PR and renamed the feature flag so there's two now :+1: